tools/editor/project_settings.cpp

```diff
@@ -81,10 +81,10 @@ void ProjectSettings::_notification(int p_what) {
 
 		translation_list->connect("button_pressed",this,"_translation_delete");
 		_update_actions();
-		popup_add->add_icon_item(get_icon("Keyboard","EditorIcons"),"Key",InputEvent::KEY);
-		popup_add->add_icon_item(get_icon("JoyButton","EditorIcons"),"Joy Button",InputEvent::JOYSTICK_BUTTON);
-		popup_add->add_icon_item(get_icon("JoyAxis","EditorIcons"),"Joy Axis",InputEvent::JOYSTICK_MOTION);
-		popup_add->add_icon_item(get_icon("Mouse","EditorIcons"),"Mouse Button",InputEvent::MOUSE_BUTTON);
+		popup_add->add_icon_item(get_icon("Keyboard","EditorIcons"),TTR("Key "),InputEvent::KEY);//"Key " - because the word 'key' has already been used as a key animation
```

Comment:
I understand what you're trying to achieve here, but I don't think it's the right way to do. We need a proper way to differentiate words that might need two different translations based on the context where they're used.